docs/en/guides/pipeline-integrations/administration/advanced-toolkit-administration.md

```diff
@@ -0,0 +1,1526 @@
+    iteration:
+        type: int
+    version:
+        type: int
+        format_spec: '03'
+    version_four:
+       type: int
+       format_spec: '04'
+       alias: version
+    timestamp:
+        type: str
+    width:
+        type: int
+    height:
+        type: int
+    channel:
+        type: str
+        filter_by: alphanumeric
+    SEQ:
+        type: sequence
+        format_spec: '04'
+    eye:
+        type: str
+
+# The paths section contains all the the key locations where files are to be stored
+# by the Toolkit Apps. Each path is made up of several keys (like {version} or {shot}) and
+# these are defined in the keys section above.
+# Toolkit apps use these paths as part of their configuration to define where on disk
+# different files should go.
+paths:
+
+    ##########################################################################################
+    # Shot pipeline / Maya
+
+    shot_root: 'sequences/{Sequence}/{Shot}/{Step}'
+
+    # define the location of a work area
+    shot_work_area_maya:
+      definition: '@shot_root/work/maya'
+      root_name: primary
+
+    # define the location of a publish area
+    shot_publish_area_maya:
+      definition: '@shot_root/publish/maya'
+      root_name: primary
+
+    # The location of WIP files
+    maya_shot_work:
+      definition: '@shot_root/work/maya/{name}.v{version}.ma'
+      root_name: primary
+
+    # The location of backups of WIP files
+    maya_shot_snapshot:
+      definition: '@shot_root/work/maya/snapshots/{name}.v{version}.{timestamp}.ma'
+      root_name: primary
+
+    # The location of published maya files
+    maya_shot_publish:
+      definition: '@shot_root/publish/maya/{name}.v{version}.ma'
+      root_name: primary
+
+    ##########################################################################################
+    # Asset pipeline / Maya
+
+    asset_root: 'assets/{sg_asset_type}/{Asset}/{Step}'
+
+    # define the location of a work area
+    asset_work_area_maya:
+      definition: '@asset_root/work/maya'
+      root_name: secondary
+
+    # define the location of a publish area
+    asset_publish_area_maya:
+      definition: '@asset_root/publish/maya'
+      root_name: secondary
+
+    # The location of WIP files
+    maya_asset_work:
+      definition: '@asset_root/work/maya/{name}.v{version}.ma'
+      root_name: secondary
+
+    # The location of backups of WIP files
+    maya_asset_snapshot:
+      definition: '@asset_root/work/maya/snapshots/{name}.v{version}.{timestamp}.ma'
+      root_name: secondary
+
+    # The location of published maya files
+    maya_asset_publish:
+      definition: '@asset_root/publish/maya/{name}.v{version}.ma'
+      root_name: secondary
+
+# The strings section is similar to the paths section - but rather than defining paths
+# on disk, it contains a list of strings. Strings are typically used when you want to be
+# able to configure the way data is written to Shotgun - it may be the name field for a
```

Comment:
Should be ShotGrid

Reply:
Fixed in latest commit, thanks @mathurf !